oidc-ui/.env

```diff
@@ -45,4 +45,13 @@ REACT_APP_WALLET_LOGO_URL=""
 
 REACT_APP_CONSENT_SCREEN_TIME_OUT_BUFFER_IN_SEC=5
 
-REACT_APP_WALLET_QR_CODE_AUTO_REFRESH_LIMIT=3
\ No newline at end of file
+REACT_APP_WALLET_QR_CODE_AUTO_REFRESH_LIMIT=3
+
+REACT_APP_LOGIN_CARD_BOX_COLOR="#BCC0C7"
```

Comment:
can we remove these values from here now?

Reply:
remove this value as well